scripts/do-release.sh

```diff
@@ -142,9 +142,12 @@ function adjust_shipyard() {
         # Make sure subctl is taken from devel, as it won't be available yet
         cd projects/shipyard
         make images IMAGES_ARGS="--buildargs 'SUBCTL_VERSION=devel'"
+        make multiarch-images IMAGES_ARGS="--buildargs 'SUBCTL_VERSION=devel'"
```

Comment:
```suggestion
        make images multiarch-images IMAGES_ARGS="--buildargs 'SUBCTL_VERSION=devel'"
```

?

(Similar to how [shipyard](https://github.com/submariner-io/shipyard/blob/devel/gh-actions/release-images/action.yaml#L28) does it)